From 2b3df5c8607ddad855a7e6b194d4028610b3a6f2 Mon Sep 17 00:00:00 2001
From: Terry Lv <r65388@freescale.com>
Date: Mon, 13 Feb 2012 19:00:48 +0800
Subject: [PATCH 449/691] ENGR00174104-3: Fix mx6 iram boot configs' issue

Two issues are fixed:
1. Enlarge malloc size to 10K.
2. Too many configs in sabresd's iram config, remove redundent configs.

Signed-off-by: Terry Lv <r65388@freescale.com>
---
 include/configs/mx6q_arm2_iram.h    |    2 +-
 include/configs/mx6q_sabresd_iram.h |   28 +---------------------------
 2 files changed, 2 insertions(+), 28 deletions(-)

diff --git a/include/configs/mx6q_arm2_iram.h b/include/configs/mx6q_arm2_iram.h
index d60d628..3a35720 100644
--- a/include/configs/mx6q_arm2_iram.h
+++ b/include/configs/mx6q_arm2_iram.h
@@ -56,7 +56,7 @@
 /*
  * Size of malloc() pool
  */
-#define CONFIG_SYS_MALLOC_LEN		(3 * 1024)
+#define CONFIG_SYS_MALLOC_LEN		(10 * 1024)
 /* size in bytes reserved for initial data */
 #define CONFIG_SYS_GBL_DATA_SIZE	128
 
diff --git a/include/configs/mx6q_sabresd_iram.h b/include/configs/mx6q_sabresd_iram.h
index 6080723..1074840 100644
--- a/include/configs/mx6q_sabresd_iram.h
+++ b/include/configs/mx6q_sabresd_iram.h
@@ -56,7 +56,7 @@
 /*
  * Size of malloc() pool
  */
-#define CONFIG_SYS_MALLOC_LEN		(3 * 1024)
+#define CONFIG_SYS_MALLOC_LEN		(10 * 1024)
 /* size in bytes reserved for initial data */
 #define CONFIG_SYS_GBL_DATA_SIZE	128
 
@@ -76,17 +76,6 @@
  * Command definition
  ***********************************************************/
 
-#include <config_cmd_default.h>
-
-#define CONFIG_CMD_PING
-#define CONFIG_CMD_DHCP
-#define CONFIG_CMD_MII
-#define CONFIG_CMD_NET
-#define CONFIG_NET_RETRY_COUNT  100
-#define CONFIG_NET_MULTI 1
-#define CONFIG_BOOTP_SUBNETMASK
-#define CONFIG_BOOTP_GATEWAY
-#define CONFIG_BOOTP_DNS
 #define CONFIG_CMD_BDI		/* bdinfo			*/
 #define CONFIG_CMD_BOOTD	/* bootd			*/
 #define CONFIG_CMD_CONSOLE	/* coninfo			*/
@@ -144,21 +133,6 @@
 
 #define CONFIG_CMDLINE_EDITING
 
-#define CONFIG_FEC0_IOBASE	ENET_BASE_ADDR
-#define CONFIG_FEC0_PINMUX	-1
-#define CONFIG_FEC0_MIIBASE	-1
-#define CONFIG_GET_FEC_MAC_ADDR_FROM_IIM
-#define CONFIG_MXC_FEC
-#define CONFIG_FEC0_PHY_ADDR		0
-#define CONFIG_ETH_PRIME
-#define CONFIG_RMII
-#define CONFIG_CMD_MII
-#define CONFIG_CMD_DHCP
-#define CONFIG_CMD_PING
-#define CONFIG_IPADDR			192.168.1.103
-#define CONFIG_SERVERIP			192.168.1.101
-#define CONFIG_NETMASK			255.255.255.0
-
 /*-----------------------------------------------------------------------
  * Stack sizes
  *
-- 
1.7.7.4

